docs/sources/whatsnew/whats-new-next/README.md

```diff
@@ -0,0 +1,57 @@
+# Contribute to 'What's New in Grafana Cloud'
+
+To have a feature presented in [What's New in Grafana Cloud](https://grafana.com/docs/grafana-cloud/whatsnew/), add an entry to the [`index.md`](./index.md) file in this directory.
+
+Use the following template, replace any `<VARIABLE>` with the appropriate text, explained later:
+
+```markdown
+## <FEATURE>
+
+<!-- <CONTRIBUTOR> -->
+<!-- <AVAILABILITY> -->
```

Comment:
Availability ends up meaning two different things based on the way we're using it here vs in the Cloud part. What about (and if we like this here, we use it on line 23 as well):
```suggestion
<!-- <ON-PREM OFFERING> -->
```